src/configProfileTemplate/config.ini

```diff
@@ -29,6 +29,10 @@ websocket_origins = [
 enabled = true
 base_path = "/"
 public_dir = "./public_http"
+status_enabled = true
+status_allowed_hosts = [
+    '127.0.0.1'
```

Comment:
This needs to allow cidr ranges and then include the local ranges by default. Eg. https://github.com/kiwiirc/webircgateway/blob/master/config.conf.example#L81

Reply:
Another quick thought - a `status_path` option would be handy too